pages/More/components/UserMenu/components/LoggedIn/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from '@shopgate/pwa-common/components/Portal';
+import * as commonPortals from '@shopgate/pwa-common/constants/Portals';
+import List from 'Components/List';
+
+/**
+ * Logged in user menu.
+ * @param {function} logout Logout handler.
+ * @returns {JSX}
+ */
+const LoggedIn = ({ logout }) => (
+  <div data-test-id="userMenu">
```

Comment:
Before the whole user menu (header + entries) was wrapped by an element with this test-id. Now it's only around the logout button.

Reply:
He's fine with it, but to make his life easier I'd add as test-id for a header.